### ISSUE TITLE
Keybinds take 2 presses

### ISSUE THREAD

**Issue by saxamaphone69** [TuxedoTako/4chan-xt#101]
Enable Keybinds, press Q or F to show Quick Reply or activate Fappe Tyme, first press is the error, second press enables as expected.

```
Uncaught TypeError: Cannot read properties of undefined (reading 'toLowerCase')
    at userscript.html?name=4chan-XT.user.js&id=10d5d7a8-76a3-49ab-a7bd-3d6108ba965d:16375:43
    at Object.keyCode (userscript.html?name=4chan-XT.user.js&id=10d5d7a8-76a3-49ab-a7bd-3d6108ba965d:16381:11)
    at HTMLDocument.keydown (userscript.html?name=4chan-XT.user.js&id=10d5d7a8-76a3-49ab-a7bd-3d6108ba965d:16076:28)
```

```
    keyCode(e) {
      let key = (() => { let kc;
      switch ((kc = e.keyCode)) {
        case 8: // return
          return '';
        case 13:
          return 'Enter';
        case 27:
          return 'Esc';
        case 32:
          return 'Space';
        case 37:
          return 'Left';
        case 38:
          return 'Up';
        case 39:
          return 'Right';
        case 40:
          return 'Down';
        case 188:
          return 'Comma';
        case 190:
          return 'Period';
        case 191:
          return 'Slash';
        case 59: case 186:
          return 'Semicolon';
        default:
          if ((48 <= kc && kc <= 57) || (65 <= kc && kc <= 90)) { // 0-9, A-Z
->            return String.fromCharCode(kc).toLowerCase();
          } else if (96 <= kc && kc <= 105) { // numpad 0-9
            return String.fromCharCode(kc - 48).toLowerCase();
          } else {
            return null;
          }
      } })();
      if (key) {
        if (e.altKey) {   key = 'Alt+'   + key; }
        if (e.ctrlKey) {  key = 'Ctrl+'  + key; }
        if (e.metaKey) {  key = 'Meta+'  + key; }
        if (e.shiftKey) { key = 'Shift+' + key; }
      }
      return key;
    },
```

On another note, I [had attempted a PR](https://github.com/ccd0/4chan-x/pull/3363) to stop using the deprecated `keyCode`, though not sure it directly translates here.

**Comment by TuxedoTako** [comment]
I can reproduce it. This is a weird one. String.fromCharCode() is supposed to be build in. How can it fail?

![msedge_vYlqtOmcU3](https://github.com/user-attachments/assets/c064894b-6f1c-4745-87d1-97529440c615)

**Comment by TuxedoTako** [comment]
Found it: String.fromCharCode is stubbed once, with `// XXX Remove document-breaking ad` as the only reason given, but even with my add blocker off, this isn't called by anything before my script.

**Comment by TuxedoTako** [comment]
Should be fixed in [2.13.3](https://github.com/TuxedoTako/4chan-xt/releases).

**Comment by saxamaphone69** [comment]
Looks fixed on my end. Thanks.
